.github/workflows/plugins-test.yaml

```diff
@@ -271,6 +271,12 @@ jobs:
         run: bash test/plugin/run.sh undertow-scenario
       - name: Run jedis 2.4.0-2.9.0 (18)
         run: bash test/plugin/run.sh jedis-scenario
+      - name: Run finagle 6.25.0-6.43.0
+        run: bash test/plugin/run.sh finagle-6.25.x-scenario
+      - name: Run finagle 6.44.0-7.1.0
+        run: bash test/plugin/run.sh finagle-6.44.x-scenario
+      - name: Run finagle 17.10.0-20.1.0
+        run: bash test/plugin/run.sh finagle-17.10.x-scenario
```

Comment:
Are these versions using different coding API for framework end users? Even from 6.33 to 6.34? Could you explain?

Reply:
I am feeling you are not using the witness mechanism properly.

Reply:
I just look your test cases, there are 3 folders but the codes are sharing or same. You should merge them into one case with version from 6.25-20.1. Because the only reason we separate the case, because the library APIs changed from version to version. But for finagle, I think it isn't. Am I right?

Reply:
> Are these versions using different coding API for framework end users? Even from 6.33 to 6.34? Could you explain?

Yes.  To use finagle, at least needs two jars, `finagle-core` and `finagle-{protocol}(eg: finagle-thrift, finagle-http)`,  I choose `finagle-thrift` in tests.  The plugin implementation is only based on `finagle-core`, but the difference of coding API comes from `finagle-thrift`.  

Specifically, to use `finagle-thrift`, `scrooge-maven-plugin` is also needed to generate code from thrift file,  and the versions of  `finagle-thrift` and `scrooge-maven-plugin` should be compatible.   In these tests, the generatd code of  `finagle-6.25.x-scenario` does not compatible with finagle-thrift 6.44.  and the generated code of `finagle-6.44.x-scenario` indeed compatible with finagle-thrift 6.43, but does not  compatible with all versions from 6.25 to 6.43.  I put these versions that both tests compatible into `finagle-6.25.x-scenario`.  And from version 17.10, these three jars(`finagle-core`,`finagle-thrift`,`scrooge-maven-plugin`) just has the same version.

Reply:
OK. Let's leave the cases like this for now. Please continue on other changes.